go.mod

```diff
@@ -3,12 +3,12 @@ module fortio.org/fortio
 go 1.16
```

Comment:
should this be changed to 1.19?

Reply:
We can sync back with upstream time to time, Fortio is currently on Go 1.18. 
![image](https://user-images.githubusercontent.com/22132944/201316429-294c2f9d-c6de-4d04-be6d-cbc4f5e606b2.png)